PayPal/src/main/java/com/braintreepayments/api/paypal/PayPalInternalClient.kt

```diff
@@ -211,7 +211,7 @@ internal class PayPalInternalClient(
     ) {
         val params = editFiParameters(payPalVaultEditRequest.editPayPalVaultId).toMutableMap()
 
-        params["risk_correlation_id"] = riskCorrelationId
+        params["correlation_id"] = riskCorrelationId
```

Comment:
Is there any risk of older versions still sending `risk_correlation_id` to the gateway? Was a server-side change recently made?

Reply:
This is for edit FI and what we are sending today as `risk_correlation_id` they are ignoring since the actual property name is supposed to be `correlation_id`. No changes were made, we just never were provided this property name in the contract. Not sure where this name was pulled from.